src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesCore.ISettable.cs

```diff
@@ -4,17 +4,32 @@ namespace BizHawk.Emulation.Cores.Nintendo.BSNES
 {
 	public partial class BsnesCore : ISettable<BsnesCore.SnesSettings, BsnesCore.SnesSyncSettings>
 	{
+		// names need to exactly match the firmware db names
+		public enum SATELLAVIEW_CARTRIDGE
+		{
+			Autodetect,
+			Rom_BSX,
+			Rom_Mahjong,
+			Rom_GNEXT,
+			Rom_RPG_Tsukuru,
+			Rom_SameGame,
+			Rom_DS96,
+			Rom_Ongaku_Tsukuru,
+			Rom_SoundNovel_Tsukuru,
+			Rom_Tsuri
+		}
+
 		public SnesSettings GetSettings()
 		{
-			return _settings.Clone();
+			return _settings with {};
```

Comment:
IIRC this is a compile-time thing, but I'll leave this reminder for myself to check it works under Mono just in case.